CATesting/config.yaml

```diff
@@ -0,0 +1,119 @@
+# Constants
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 10}}
+{{$qps := DefaultParam .qps 10}}
```

Comment:
qps is inconsistent with other environment variables naming like PODS_PER_NODE

Reply:
Sounds good.